src/promptflow/promptflow/_sdk/_service/entry.py

```diff
@@ -78,7 +78,7 @@ def validate_port(port, force_start):
                 app.logger.warning(f"Force restart the service on the port {port}.")
                 kill_exist_service(port)
             else:
-                app.logger.warning(f"Service port {port} is used.")
+                app.logger.warning(f"Service port {port} is used, version: {get_promptflow_sdk_version()}.")
```

Comment:
should get version from the service return response?

Reply:
you mean heattbeat api? I doubt there're scenario that port is used but service is not healthy.

Reply:
The scenario I want to print version, is that user is using an old version of PFS. which could not be aware by user. 
As the user can do pip install promptflow, but forget to pfs start.